tests/support/server.rs

```diff
@@ -23,6 +23,7 @@ pub struct Txn {
     pub read_timeout: Option<Duration>,
     pub response_timeout: Option<Duration>,
     pub write_timeout: Option<Duration>,
+    pub chunk_size: Option<usize>,
```

Comment:
Additionally, you will get interesting results if you sent it with *chunk encoding* of size 1. It should break because read_exact is not transactional (see above).